README.md

```diff
@@ -4,6 +4,30 @@
 
 This is an API for managing load balancer configurations for the Infratographer platform.
 
+## Load Balancer Components
```

Comment:
This will change when it gets renamed to load balancers, ports, pools and origins but I wanted to write it down so it was easier to understand.